internal/commands/vaultsecrets/secrets/versions.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package secrets
+
+import (
+	"context"
+	"fmt"
+
+	preview_secret_service "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/preview/2023-11-28/client/secret_service"
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/preview/2023-11-28/models"
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/stable/2023-06-13/client/secret_service"
+
+	"github.com/hashicorp/hcp/internal/pkg/cmd"
+	"github.com/hashicorp/hcp/internal/pkg/format"
+	"github.com/hashicorp/hcp/internal/pkg/heredoc"
+	"github.com/hashicorp/hcp/internal/pkg/iostreams"
+	"github.com/hashicorp/hcp/internal/pkg/profile"
+)
+
+func NewCmdVersions(ctx *cmd.Context, runF func(*VersionsOpts) error) *cmd.Command {
```

Comment:
Shouldn't this be a subcommand since there is CRUD on versions?

https://github.com/hashicorp/cloud-vault-secrets/blob/main/proto-public/hashicorp/cloud/secrets_20231128/secrets.proto#L105-L151

As in `hcp vault-secrets secrets versions [list, get, delete, access]`

@briankassouf

Reply:
Yep! it should be a subcommand with those CRUD operations under it

Reply:
In particular this functionality should be under `hcp vault-secrets secrets versions list`

Reply:
@xka5h thoughts on pivot to proposed command structure vs what we have decided till now?

Reply:
@xka5h and I chatted on the proposal here, and he has accepted it. I am going to rework things in my next commit. thansk!